### PR TITLE
Update Flickable and ScrollView docs

### PR DESCRIPTION
**Pull request by jrmoulton**
Include documentation about needing to manually set the viewport-width and viewport-height when using a for loop.
I also added some more general documentation to `Flickable` but it likely needs to be edited as my explanation was a bit wordy. 
Are there other points in the documentation that need this same explanation or is this sufficient?

### CODE REVIEW

File: docs/builtin_elements.md [hunk 1]
```diff
@@ -578,6 +578,18 @@ FIXME: write docs
 
 FIXME: write docs
```
Comment (ogoffart) [inline docs/builtin_elements.md:579]: I guess this line can go away 🙂

File: docs/builtin_elements.md [hunk 2]
```diff
@@ -578,6 +578,18 @@ FIXME: write docs
...
 
+The `Flickable` is a lower-level item that is the base for scrollable
+elements, such as the ScrollView widget. When the viewport-width or the
+viewport-height is greater than the parent width or parent height
+respectively the element becomes scrollable although the `Flickable`
+does not create a scrollbar. When using a for loop to populate the
```
Comment (ogoffart) [inline docs/builtin_elements.md:585]: But one should say that it does automatically computes it based on its contents. With the exception of the bug if it is a for loop

**Review by ogoffart — APPROVED** [review]
Great, thanks.

It would be nice to have a "Properties" and "Example" section, like for the other items.
The properties are listed there: https://github.com/sixtyfpsui/sixtyfps/blob/cd47ef11005512bfd7799ce16a98dd4c71186a41/sixtyfps_compiler/builtins.60#L150-L161

But I actually can do that myself next week after meeting

**Comment by ogoffart** [comment]
I don't think the CI failure is related to this change.

**Comment by jrmoulton** [comment]
I added some of those changes. I didn't add all of the properties to Flickable because it looked like similar properties were omitted in the docs on the other elements as well. But if needed I can add those back in. I also didn't add descriptions one some properties because I'm not exactly sure what their functions are.

**Comment by ogoffart** [comment]
Good enough, thank you!
I'll just do some adjustments after merging
